How-to-Install-SCF.md

```diff
@@ -207,8 +216,11 @@ To install SCF
   To invoke the tests run the command
    ```
    kubectl create \
-      --namespace="${NAMESPACE}" \
+      --namespace=scf \
       --filename="kube/cf/bosh-task/smoke-tests.yml"
```

Comment:
Nope. (Unless you count doing it with pipes and `kubectl create -f -` as not changing the file.)